presto-main/src/test/java/io/prestosql/cost/AbstractTestStatsCalculator.java

```diff
@@ -16,7 +16,7 @@
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 
-public abstract class BaseStatsCalculatorTest
+public abstract class AbstractTestStatsCalculator
```

Comment:
I didn't know that was a convention. I see nothing wrong with the base class being called `BaseXXX`. In fact, I prefer it over `AbstractXXX`, which sounds very Hungarian notation-like. "Base" also more accurately conveys the *purpose* of the class vs "Abstract", which describes one of its physical characteristics.

Reply:
I also thought that `Base.*Test` is a new prefered convention over `Abstract.*`.

Reply:
Applied